website/registrations/migrations/0012_remove_registration_project_registration_projects.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 4.1.3 on 2024-09-06 11:59
+
+from django.db import migrations, models
+
+
+def migrate_projects(apps, schema_editor):
+    Registration = apps.get_model("registrations", "Registration")
+
+    for registration in Registration.objects.all():
+        registration.projects.add(registration.project)
```

Comment:
I am not sure how this adds the current project registration to the new m2m table, are you sure we are not loosing data here?

Reply:
First, we add a new m2m field "projects" (while the old fk field project still exists). Then we add the current project to the set of projects, and only afterwards we remove the project fk field. 

So I guess that shouldn't remove data right?

Or am I overlooking some obvious mistake 🤔

Reply:
Yeah I guess I overlooked the relation between registrations and project(s). Looks fine